saleor/graphql/shop/mutations.py

```diff
@@ -143,6 +144,13 @@ class Meta:
         error_type_class = ShopError
         error_type_field = "shop_errors"
 
+    @classmethod
+    def check_permissions(cls, context, permissions=None):
+        # for cloud instances this mutation is disabled and raises PermissionDenied
+        if settings.IS_CLOUD_INSTANCE:
```

Comment:
We should avoid implementing cloud logic in saleor core. We have a separate repository with saleor's cloud build and all cloud-related features. I'll create a patch for cloud build to handle this case. Please close this PR as not needed in core itself.

Reply:
OK, I'm closing this PR then.